modules/appeals_api/app/services/appeals_api/pdf_construction/generator.rb

```diff
@@ -7,7 +7,7 @@ module PdfConstruction
     class Generator
       def initialize(appeal, pdf_version: 'v1')
         @appeal = appeal
-        appeal.update(pdf_version:)
+        appeal.update(pdf_version:) if appeal.pdf_version.blank?
```

Comment:
This method is called by the download endpoint to generate a PDF before watermarking. In that case, the appeal will already have a `pdf_version` set, so I was thinking it wouldn't make sense to update it here.